### ISSUE TITLE
OSnet for detection and Re-identification

### ISSUE THREAD

**Issue by BurhanSabir** [KaiyangZhou/deep-person-reid#450]
Does OSnet model alone can handle detection and re-identification? 
If Yes, then how?
Or if we require to use other model what it will be any suggestions?

**Comment by LamnouarMohamed** [comment]
Torchreid just for person re-identification, you can use other model for object detection like SSD, or Faster RCNN, or Yolo for extract the bounding box.

**Comment by BurhanSabir** [comment]
Yup, I got that and now I am using yolov3 for tracking purposes and osnet for re-id, anyways thank you mate

**Comment by LamnouarMohamed** [comment]
You're welcome any time
